packages/client-utils/src/sync-tools.js

```diff
@@ -287,3 +288,46 @@ export const waitUntilInvitationReceived = (addr, io, options) => {
     { reusePromise: true, setTimeout, ...resolvedOptions },
   );
 };
+
+/// ////////// Making sure an offer was exited successfully /////////////
+
+const makeQueryWalletCurrent = follow => async (/** @type {string} */ addr) => {
+  const update = await follow('-lF', `:published.wallet.${addr}.current`);
+
+  return update;
+};
```

Comment:
I don't think there is any difference between the above and this shorter version.
```suggestion
const makeQueryWalletCurrent = follow => (/** @type {string} */ addr) =>
  follow('-lF', `:published.wallet.${addr}.current`);
```

Reply:
Change request addressed